src/mixins/legal-api-mixin.ts

```diff
@@ -165,4 +137,35 @@ export default class LegalApiMixin extends Vue {
         throw error
       })
   }
+
+  /**
+    * Ensure consisent object structure for an incorporation application
+    * whether it contains a Name Request or not, and whether it is an initial
+    * draft or it has been previously saved. Object merging does not
+    * work very well otherwise (due to nested properties)
+    * @param filing The filing fetched from legal-api
+    * @returns the filing in safe-empty state if applicable
+  */
+  formatEmptyFiling (filing: any): IncorporationFilingIF {
```

Comment:
Would the back end return empty drafts in different structures or wouldn't return the properties at all? I'm just curious about this methods purpose?

Reply:
an "empty" draft does not have the same format as a FE filing object...this is a way of ensuring if it is missing necessary fields, that we have a safe-empty object to work with (this would only be an issue for new drafts, otherwise this method passes through the original filing that was retrieved)

Reply:
I see i see, fair enough. Thanks!